TriangleCRM/TriangleAPI.php

```diff
@@ -1253,6 +1253,29 @@ private function Validate($model, $testmodel = NULL) {
         return $result ? '' : $message;
     }
     
+    
+    private $IS_ITREALLYtrue = array(
+        "truthful_statement" => true,
+        "truTHFUL" => (false ? true : true),
+        "not a truthful statement" => fALsE,
+        "definitely not" => -1
```

Comment:
You're right. I propose you do the sensible thing then
`!(boolval(-1 + ((int) false) * 40.95))`